src/Organic/SDK/OrganicSdk.php

```diff
@@ -495,27 +495,41 @@ public function queryAffiliateConfig() {
     }
 
     /**
+     * Run a mutation on the platform API to save important WordPress settings platform-side,
+     * then query the response to save important platform settings here on the WordPress side.
+     *
      * @throws RuntimeException
      */
-    public function queryWordPressConfig() {
-        // Make a call to platform API for client-specific plugin config.
-        $gql = new Query( 'wordpressPluginConfig' );
-        $gql->setArguments(
-            [
-                'siteGuid' => $this->siteGuid,
-            ]
+    public function mutateAndQueryWordPressConfig( \Organic\Organic $organic ) {
+        global $wp_version;
+        $mutation = ( new Mutation( 'syncWordpressPluginConfig' ) );
+        $mutation->setVariables( [ new Variable( 'configInput', 'WordpressPluginConfigInput', true ) ] );
+        $mutation->setArguments( [ 'configInput' => '$configInput' ] );
+        $mutation->setSelectionSet(
+            [ ( new Query( 'config' ) )->setSelectionSet( [ 'sentryDsn' ] ) ]
         );
-        $gql->setSelectionSet(
-            [
-                'sentryDsn',
-            ]
-        );
-        try {
-            $result = $this->runQuery( $gql );
-        } catch ( \Exception $e ) {
-            throw new RuntimeException( 'Failed to fetch WordPress Plugin config' );
-        }
-        return $result['data']['wordpressPluginConfig'];
+
+        # Note that this might be true even before the site is fully migrated.
+        $organicContentEnabled = function_exists( '\SWPCore\HeadlessFrontend\isAvailable' );
+
+        $variables = [
+            'configInput' => [
+                'siteGuid' => $this->siteGuid,
+                'organicIntegrationEnabled' => $organic->isEnabled(),
+                'organicContentEnabled' => $organicContentEnabled,
+                'phpVersion' => phpversion(),
+                'wordpressVersion' => $wp_version,
+                'pluginVersion' => $organic->version,
+                'sdkVersion' => $organic->getSdkVersion(),
+                'adsTxtRedirectEnabled' => (bool) $organic->getOption( 'organic::ads_txt_redirect_enabled' ),
```

Comment:
This returns an int. I wasn't sure there was a reason this was not exposed on $organic (e.g., `$organic->adsTextRedirectEnabled()`), so I left it alone just in case.